src/sentry/integrations/github/webhook.py

```diff
@@ -69,6 +69,7 @@ def __call__(self, event: Mapping[str, Any], host: str | None = None) -> None:
                     "external_id": str(external_id),
                 },
             )
+            logger.exception()
```

Comment:
these might need a string? I don't see any other places in the code where we call .exception with no string

Reply:
i also don't know how logger works

Reply:
It doesn't work. I have fixed it.